codestodians/komlie-frontend
# [Feature] Create cart page

#### What does this PR do?
- Creates a page to display cart items

#### Description of Task to be completed?
- Create cart route, mount cart component on cart route.
- Deprecate cart dropdown from cart icon.

#### How should this be manually tested?
- Start the app
- Click on the cart icon
- You should be navigated to the cart page 

#### What are the relevant pivotal tracker stories?
- [#164009443](https://www.pivotaltracker.com/story/show/164009443)

#### Screenshots (if appropriate)
![Screen Shot 2019-06-21 at 2 32 10 PM (2)](https://user-images.githubusercontent.com/22825700/59926610-92949f00-9432-11e9-888f-c7b91055a686.png)
![Screen Shot 2019-06-21 at 2 31 59 PM (2)](https://user-images.githubusercontent.com/22825700/59926661-b0fa9a80-9432-11e9-916f-4e7c5a05423d.png)


#### Additional Info
- The items in the cart at the moment are dummy items. If the items array is empty, the cart page displays an empty cart icon. My subsequent work will be on adding a product to the cart so that actual products are displayed as opposed to the current dummy items.
